packages/eslint-plugin-orbit-components/src/rules/arrowFunctionsCount.ts

```diff
@@ -0,0 +1,48 @@
+import * as t from "@babel/types";
+import type { Rule } from "eslint";
+
+const LIMIT = 5;
```

Comment:
I wonder if the limit of five is too arbitrary 🤔  Maybe we should be more opinionated and simply warn against more than one?

Reply:
Having it set up for just one will be very limiting. The common use case is to have some styles together with media queries. If we would count also with more complex interpolations I think that limiting it to `~5` is fine. Alternatively we could have an option for the rule with defining the `schema` in meta.